### PR TITLE
[Doc]Update config example and bump to v6.0.3

### PR DESCRIPTION
**Pull request by karenzone**
Screen Shot 2019-10-02 at 4 09 05 PM

### CODE REVIEW

**Comment by karenzone** [comment]
Note to self (@karenzone):  The version number is currently hardcoded in the example. Follow up with #371 to see if this is a viable solution.

**Comment by karenzone** [comment]
@robbavey The versioning mechanism is a bit different from what I've seen before. Will you please help keep me honest on the version bump in the gemspec?

File: docs/index.asciidoc [hunk 1]
```diff
@@ -37,21 +38,23 @@ input {
...
-Logstash 6.0 or later, do not set `document_type` in your configuration because
-Logstash sets the type to `doc` by default.
+-----
+<1> `%{[@metadata][beat]}` sets the first part of the index name to the value
+of the `beat` metadata field and `%{[@metadata][version]}` sets the second part to
+the Beat's version. For example:
+metricbeat-7.4.0.
+
+Events indexed into Elasticsearch with the Logstash configuration shown here
+will be similar to events directly indexed by beats into Elasticsearch.
```
Comment (dedemorton) [inline docs/index.asciidoc]:
```suggestion
will be similar to events directly indexed by Beats into Elasticsearch.
```

**Comment by karenzone** [comment]
**Published 2019-1003, Thu**
Screen Shot 2019-10-03 at 3 52 32 PM

**Pull request merged**